modules/storages/lib/api/v3/storages/storage_representer.rb

```diff
@@ -90,14 +90,28 @@ def link_without_resource(name, getter:, setter:)
                end
              }
 
+    property :tenant_id,
+             skip_render: ->(represented:, **) { !represented.provider_type_one_drive? },
```

Comment:
Would the following have the same behavior? If yes, then I suggest to rewrite it like that. It is simpler to read/understand imho.
```suggestion
             if: ->(represented:, **) { represented.provider_type_one_drive? },
```

Reply:
As far as I understand, this is not the same. Especially, as `skip_render` explicitly targets only rendering, and not parsing. parsing is unaffected, so that I can write the `tenant_id` always. It will be simply ignored by any storage creation of type nextcloud.

This is a pattern I at least found at other places in our representers.

Reply:
Okay, I see. Thanks for clarifying!